.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
-.env
```

Comment:
ah, I see you had it in your git ignore but removed it, for good practice, it should always be ignored